### PR TITLE
Make price inquirer also work with WETH

### PR DESCRIPTION
**Pull request by coinyon**
This renames WETH to ETH for cryptocompare

### CODE REVIEW

File: rotkehlchen/constants/cryptocompare.py [hunk 1]
```diff
@@ -280,6 +280,8 @@
     # For Rotkehlchen WEB is Webcoin and WEB-2 Webchain, but in cryptocompare
     # Webchain is WEBC
     'WEB-2': 'WEBC',
+    # Cryptocompare does not have WETH, so report price for ETH
+    'WETH': 'ETH',
```
Comment (LefterisJP) [inline rotkehlchen/constants/cryptocompare.py:284]: Hmm `WETH` and `ETH` should be interchangeable so this should be fine but it's also weird.

Cryptocompare does have `WETH`: https://www.cryptocompare.com/coins/weth/overview

But it seems that indeed no price is actually reported.

Random query with `WETH`: https://min-api.cryptocompare.com/data/pricehistorical?fsym=WETH&tsyms=EUR&ts=1584439980
response: `{"WETH":{"EUR":0}}`

Same query with `ETH`: https://min-api.cryptocompare.com/data/pricehistorical?fsym=ETH&tsyms=EUR&ts=1584439980
response: `{"ETH":{"EUR":106.44}}`

Noteworthy that `WBTC` works fine, and actually has different price than BTC. Same timestamp query:

https://min-api.cryptocompare.com/data/pricehistorical?fsym=WBTC&tsyms=EUR&ts=1584439980
response: `{"WBTC":{"EUR":5510.4}}`
https://min-api.cryptocompare.com/data/pricehistorical?fsym=BTC&tsyms=EUR&ts=1584439980
response: `{"BTC":{"EUR":4876.52}}`

Comment (coinyon) [reply rotkehlchen/constants/cryptocompare.py:284]: Yea its weird. I didn't get 0.0 but this error:

    ERROR Cryptocompare usd price query failed asset=WETH

Comment (LefterisJP) [reply rotkehlchen/constants/cryptocompare.py:284]: Yes since we use the histohour endpoint, but data should be similar.

Comment (apiha-cryptocompare) [reply rotkehlchen/constants/cryptocompare.py:284]: Hello all.
CryptoComapre has WETH as a coin listed. For pair WETH_ETH we integrate Uniswap exchange:

https://min-api.cryptocompare.com/data/pricehistorical?fsym=WETH&tsyms=ETH&ts=1594109179&e=uniswap

https://www.cryptocompare.com/coins/weth/trades/ETH

"pricehistorical" endpoint has parameter "e" which stands for exchange (https://min-api.cryptocompare.com/documentation?key=Historical&cat=dataPriceHistorical). 
Default value is "CCCAGG" which is our own index.
Currently pair WETH_ETH is not in our index. 
We have a review process for each pair. After it's reviewed it could be added to CCCAGG.

Hope that helps.
Kind Regards.

Comment (LefterisJP) [reply rotkehlchen/constants/cryptocompare.py:284]: Hey @apiha-cryptocompare thanks a lot for coming in here and explaining.

From our side it's really of no relevance what exchange the price is pulled from. I expect some kind of average of all exchanges from which a price can be pulled of.

When a pair is queried we really can't specify exchanges to query from per pair.

Can't this endpoint detect that your index has no price for it and fallback to uniswap? If a price exists for something the endpoint should return the price and not return 0. That's what a user would expect.

Perhaps you can make the price endpoints work like that? We simply want a price, and don't care where it's coming from. The more exchanges (for an average of all prices) the better, but if no exchange arguments is specified and an exchange has a price it would be great to get that. As is the case with our problem here.

**Comment by LefterisJP** [comment]
I am merging since the failing test is a travis issue on OSX: https://github.com/rotki/rotki/issues/1131

Let's use this hack until cryptocompare fixes the way they return the prices for pairs they have a price for but are not in their index.

File: rotkehlchen/constants/cryptocompare.py (hunk 1 above)
Comment (LefterisJP) [reply rotkehlchen/constants/cryptocompare.py:284]: Hey again  @apiha-cryptocompare disregard my suggestion above. I had a chat with Vlad about it and I understand why it's not as easy as I thought.


Tl;DR: Many exchanges have fake volume and prices and many of them have different tokens listed for same symbols so mapping need to be maintained. That's why manual work is needed.

Again thank you for the explanation and coming all the way here to help us :hugs:

Comment (apiha-cryptocompare) [reply rotkehlchen/constants/cryptocompare.py:284]: Hey @LefterisJP 

No problem at all, we're happy to help If we can. 

Regarding this particular pair WETH_ETH we'll include it and regenerate in our next CCCAGG review.

If it's of any help, you can check and subscribe for CCCAGG reviews here: 
https://data.cryptocompare.com/reports/cryptocompare-aggregate-index-constituent-review

Thank you and keep up the good work with this amazing product of yours :+1:
